src/styles/main-styles.scss

```diff
@@ -163,14 +163,18 @@ pre {
 
 header{
   .main &.header{ padding: 0 };
+
+  .navbar-collapse{
+    background: $green !important;
+  }
   
   .main-nav {
     @media(min-width: 968px){
       float: right;
     }
 
     .nav{
-      padding-top: 60px;
+      // padding-top: 60px;
```

Comment:
Why a comment? Please remove it.

Reply:
lol... didn't realized you actually reviewed this stuff. my apologies, i'll start paying attention.